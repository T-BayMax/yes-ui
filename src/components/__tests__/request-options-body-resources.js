import React from 'react';
import Resource from '../request/request-options-body-resources';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';

test('Link changes the class when hovered', () => {
    let resourceButton = false
    var resourceClose = () =>{

    }
    const component = renderer.create(
        <Resource resourceClose={resourceClose} resourceButton={true} />
    );
    // const aa = shallow(
    //     <Resource resourceClose={resourceClose} resourceButton={true} />
    // );
    // expect(aa).toMatchSnapshot()
    component.toJSON().props.openDialog()
    expect(component.toJSON()).toMatchSnapshot()

    // expect(component.toJSON()).toMatchSnapshot()

});