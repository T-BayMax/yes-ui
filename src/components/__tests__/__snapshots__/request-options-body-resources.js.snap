// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
Object {
  "className": "Resource-wrapper-1",
  "openDialog": [Function],
}
`;
